menu.xml

```diff
@@ -11,25 +11,25 @@
   <OXMENU id="NAVIGATION_ESHOPADMIN">
     <MAINMENU id="mxorders">
       <SUBMENU id="mxdisplayorders" cl="admin_order" list="order_list">
-        <TAB id="text_list" pg_translate="1" cl="wcpg_order_transactions"/>
-        <TAB id="transaction_details_title" pg_translate="1" cl="wcpg_order_transaction_details"/>
+        <TAB id="wdpg_text_list" pg_translate="1" cl="wcpg_order_transactions"/>
```

Comment:
You could now remove the custom attribute `pg_translate="1"` from all XML files.

This was used till now to determine the keys used by the module. Instead, you can now parse all keys and filter out those that don't start with `wdpg_`.